src/Transfer.php

```diff
@@ -3986,6 +3989,12 @@ class='btn btn-primary'></div>";
         Dropdown::showFromArray('keep_disk', $keep, $params);
         echo "</td></tr>";
 
+        echo "<tr class='tab_bg_1'>";
+        echo "<td>" . __('Disable locked field during transfer') . "</td><td>";
+        Dropdown::showYesNo('locked_field', $this->fields['locked_field']);
```

Comment:
I had to read code twice to understand how field was used. I think it will be more understandable like this:
```suggestion
        echo "<td>" . __('Lock fields updated during transfer') . "</td><td>";
        Dropdown::showYesNo('lock_updated_fields', $this->fields['lock_updated_fields']);
```

Reply:
agree with that